Index: b/src/Makefile.in
===================================================================
--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -16,9 +16,9 @@ HEADREV	:= "$Revision: 2457 $"
 VERSION	= $(shell  perl -e '($$r) = (q( $(HEADREV) ) =~ m:(\d+):); $$t= q( $(HEADURL) ) =~ m:/tags/([^/]+): ? $$1 : "trunk"; print "$$t:$$r\n";' )
 
 CFLAGS	:= @CFLAGS@ @NEED_FNESTED_FUNCTIONS@
-CFLAGS	+= -Wall -funsigned-char -Os -DFSVS_VERSION='"$(VERSION)"'
+CFLAGS	+= -fgnu89-inline -Wall -funsigned-char -DFSVS_VERSION='"$(VERSION)"'
 LDFLAGS	:= @LDFLAGS@
-FSVS_LDFLAGS = $(LDFLAGS) -lsvn_subr-1 -lsvn_delta-1 -lsvn_ra-1 -lpcre -lgdbm -ldl 
+LIBS = -lsvn_subr-1 -lsvn_delta-1 -lsvn_ra-1 -lpcre -lgdbm -ldl 
 EXTRALIBS	:= @EXTRALIBS@
 WAA_CHARS?= @WAA_WC_MD5_CHARS@
 
@@ -145,14 +145,14 @@ docs: $(MANDEST)/fsvs-howto-backup.5 $(M
 
 ################################ Rules ######################################
 %.o:	%.c
-	@echo "     CC $<"
-	@$(CC) $(CFLAGS) -c -o $@ $<
+#	@echo "     CC $<"
+	$(CC) $(CFLAGS) -c -o $@ $<
 
 # if the Makefile has changed, the output will (at least sometimes)
 # change, too.
 $(DEST): $(C_FILES:%.c=%.o)
-	@echo "     Link $@"
-	@$(CC) $(FSVS_LDFLAGS) $(LDLIBS) $(LIBS) $(EXTRALIBS) -o $@ $^
+#	@echo "     Link $@"
+	$(CC) $(LDFLAGS) -o $@ $^ $(LDLIBS) $(LIBS) $(EXTRALIBS)
 ifeq (@ENABLE_RELEASE@, 1)
 	-strip $@
 endif
